misk/src/main/kotlin/misk/web/interceptors/RequestLoggingInterceptor.kt

```diff
@@ -131,7 +134,11 @@ class RequestLoggingInterceptor internal constructor(
     val sampling = if (isError) errorBodySampling else bodySampling
     val randomDouble = random.current().nextDouble()
     if (randomDouble < sampling) {
-      val requestResponseBody = bodyCapture.get()
+      val requestResponseBody =
+        requestLoggingTransformers.foldRight(bodyCapture.get()) { transformer, body ->
+          transformer.tryTransform(body)
+        }
```

Comment:
It'd be nice to see a test for behaviour with multiple transformers applied to the same request. I have a feeling there might be a gotcha with ordering these transformations?